src/testvers.cpp

```diff
@@ -0,0 +1,31 @@
+#include "semantic_vers.h"
```

Comment:
Tests are great!

Reply:
Yes... but there is still no place like test/ or so where to place it. And,  more than this is actually needed to motivate it. Resolving for now, but it is on my  TODO-list.